### PR TITLE
Fix duplicated footer with handler

### PR DESCRIPTION
**Pull request by rokatyy**
Jira - https://iguazio.atlassian.net/browse/ML-5818

`functionSourceCode` is generated using `mlrun.code_to_function` which always adds following footer, so there is no need in in duplicating it in function code definition. 

```
from mlrun.runtimes import nuclio_init_hook
def init_context(context):
    nuclio_init_hook(context, globals(), '{}')

def handler(context, event):
    return context.mlrun_handler(context, event)
 ```

### CODE REVIEW

File: tf2_serving_v2/function.yaml [hunk 1]
```diff
@@ -26,14 +26,14 @@ spec:
       annotations:
         nuclio.io/generated_by: function generated from /home/kali/functions/tf2_serving_v2/tf2_serving_v2.py
     spec:
-      runtime: python:3.6
+      runtime: python:3.8
```
Comment (TomerShor) [inline tf2_serving_v2/function.yaml:29]: why not 3.9 ?

Comment (rokatyy) [reply tf2_serving_v2/function.yaml]: @TomerShor removed this change

**Review by TomerShor — APPROVED** [review]

**Comment by Eyal-Danieli** [comment]
@rokatyy can you please also remove the footer from `churn_server` func? 
https://github.com/mlrun/functions/blob/master/churn_server/churn_server.py#L47

**Comment by Eyal-Danieli** [comment]
In addition, please raise the function ver in each `item.yaml`. For example update `v2_model_server` from `1.1.0` to `1.2.0`.

**Comment by rokatyy** [comment]
@Eyal-Danieli Sure - done

**Review by Eyal-Danieli — APPROVED** [review]

**Pull request merged**